### ISSUE TITLE
mpd.py module is not compatible with Python3

### ISSUE THREAD

**Issue by aaronchantrill** [NaomiProject/Naomi#167]
## Description

If you install the mpd.py library for python3 directly via "pip3 install python-mpd", there is an incompatibility on line 399 where the line "except socket.error, err:" should be "except socket.error as err:". This is causing the mpdcontrol plugin to fail.

## Expected Behavior

The mpd.py module should load and the mpdcontrol plugin should load:
```terminal
DEBUG:naomi.pluginstore:Found plugin candidate at: /home/pi/Naomi/plugins/speechhandler/mpdcontrol
DEBUG:naomi.pluginstore:Plugin info file '/home/pi/Naomi/plugins/speechhandler/mpdcontrol/plugin.info' parsed successfully!
DEBUG:naomi.pluginstore:Found valid plugin: mpdcontrol 1.0.0
```

## Actual Behavior

```teminal
DEBUG:naomi.pluginstore:Found plugin candidate at: /home/pi/Naomi/plugins/speechhandler/mpdcontrol
DEBUG:naomi.pluginstore:Plugin info file '/home/pi/Naomi/plugins/speechhandler/mpdcontrol/plugin.info' parsed successfully!
WARNING:naomi.pluginstore:Plugin at '/home/pi/Naomi/plugins/speechhandler/mpdcontrol' skipped! (Reason: invalid syntax)
Traceback (most recent call last):
  ...
  File "~/.local/lib/python3.5/site-packages/mpd.py", line 399
    except socket.error, err:
                       ^
SyntaxError: invalid syntax
```

## Possible Fix

There are three possible ways this could be resolved:
1. contacting the python-mpd package maintainers and report the error
2. add a fixed copy of the module to the plugin
3. use a different module altogether

## Steps to Reproduce

1. Setup and install Naomi
2. Use pip3 to install the python-mpd package
3. Start Naomi (start in debug mode to see error message)

## Context

This bug makes it impossible to set up mpd control without digging into the source code. The MPD Control plugin is an import demonstration of how to use a custom dictionary within a plugin.

## Your Environment

* Version used: naomi-dev
* Environment name and version (e.g. PHP 5.4 on nginx 1.9.1): Python 3
* Server type and version: Raspberry Pi 3B+
* Operating System and version: Raspbian Stretch

**Comment by redragonx** [comment]
I suggest we do 1 as it promotes development and improvements. Maybe do a PR?

2. would work but we would have to backport fixes later on. The team isn't big enough to do that yet.

**Comment by aaronchantrill** [comment]
I finally got around to figuring how to get mpd set up so I could test this from within Naomi. Looks like the python-mpd2 package is a direct replacement for the mpd library. It is still referenced as "import mpd" so the plugin doesn't have to change at all. It appears to me that all we need to do is update the python_requirements.txt file and change "python-mpd==0.3.0" to "python-mpd2".

**Comment by aaronchantrill** [comment]
Fixed in PR #189 by removing python-mpd and adding python-mpd2 as requirement.
